packages/CMS/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@canner/cms",
+  "version": "1.4.4",
+  "description": "Canner CMS",
```

Comment:
More accurate description in `package.json`

Reply:
I've updated the description.